synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1985,7 +1997,7 @@ CanvasView::create_tab_label()
 	label->show();
 
 	closebutton = manage(new Gtk::Button());
```

Comment:
While you are at this I'd appreciate if you could improve `closebutton` right margin (`set_margin_end()`) to value 4

Because we have this incomplete button display
![Screenshot from 2020-10-14 07-01-27](https://user-images.githubusercontent.com/24214509/95933564-7f0e0400-0dec-11eb-827c-7a02700f131b.png)

Reply:
Fixed. Didn't happen on my gtk theme.